scripts/review-dashboard/dinghy.yaml

```diff
@@ -0,0 +1,35 @@
+---
+digests:
+
+  - digest: microshift.html
+    title: Microshift Developer Dashboard
+
+    # Summarize the last day's activities in all queries
+    since: 1 day
+
+    items:
+
+      # Look for enhancement PRs that mention MicroShift
+      - title: Enhancements
+        search: repo:openshift/enhancements is:pr is:open microshift
+
+      # Include all pull requests against MicroShift itself
+      - title: Code Changes
+        url: https://github.com/openshift/microshift
+
+      # Include changes to MicroShift CI
+      - title: CI Changes
+        search: ( repo:openshift/origin OR repo:openshift/release ) is:pr is:open microshift
+
+      # Look for documentation changes for MicroShift
+      - title: Documentation
+        search: repo:openshift/openshift-docs is:pr is:open microshift
```

Comment:
I would include `microshift OR ushift` in the search query, just in case.

Reply:
Good point, done.